TombEngine/Scripting/Internal/TEN/Objects/ObjectsHandler.h

```diff
@@ -64,6 +64,45 @@ class ObjectsHandler : public ScriptInterfaceObjectsHandler
 			return std::make_unique<R>(std::get<R::IdentifierType>(m_nameMap.at(name)));
 	}
 
+	template <typename R, char const* S>
```

Comment:
You can change this to `template <typename R, char const* S>` as `S` isn't used

Reply:
done